src/main/java/org/testng/reporters/jq/Main.java

```diff
@@ -18,10 +18,11 @@
 
 public class Main implements IReporter {
   private static final String[] RESOURCES = new String[] {
-    "jquery-1.7.1.min.js", "testng-reports.css", "testng-reports.js",
-    "passed.png", "failed.png", "skipped.png", "navigator-bullet.png",
-    "bullet_point.png", "collapseall.gif"
```

Comment:
@bmorganatlas - I think this change will work fine in a non windows machine, but the moment we try this on a windows machine, I have a gut feel that things will break because of the file separator (`/` vs `\`). I dont have a windows machine right now at my disposal to vet this out.